docs/development/release_management.rst

```diff
@@ -42,27 +42,49 @@ Pre-Release
 
      securedrop/bin/dev-shell ../update_version.sh 0.x.y~rcN
 
-8. Build Debian packages and place them on ``apt-test.freedom.press``.
-9. Write a test plan that focuses on the new functionality introduced in the release.
-   Post for feedback and make changes based on suggestions from the community.
-10. Encourage QA participants to QA the release on production VMs and hardware. They
+8. If you would like to sign the release commit, you will need to do so manually:
+     8.1. Create a new signed commit and verify the signature:
+
+      ::
+
+         git reset HEAD~1
+         git commit -a
```

Comment:
`git commit -aS`? For those that have not got `gpgsign = true` in their git config